Resources/Lang/en_US.json

```diff
@@ -791,6 +793,7 @@
   "EvilMiniInfoLong": "(Impostors):\nAs the Evil Mini, you are unkillable until you grow up and have a very long initial kill cooldown, which gets drastically shortened as you grow up.",
   "BlackmailerInfoLong": "(Impostors):\nAs the Blackmailer, when you shift into a target, you will blackmail that player. This means that during the meetings, they won't be able to speak.\n\nNote: If someone is already blackmailed, blackmailing another person un-blackmails the current person.",
   "InstigatorInfoLong": "(Impostors):\nAs the Instigator, it's your job to turn the crewmates against each other. Each time a Crewmate gets voted out in a meeting, if you are alive, an additional Crewmate who voted for the innocent player will die after the meeting. The Host determines the number of additional players dying.",
+  "ExorcistInfoLong": "(Impostors):\nAs the Exorcist, you can use /ex in meeting to exorcise a player who won't shut up. The exorcised player will die. You'll lose your ability if you excersise too many people.",
```

Comment:
"ExorcistInfoLong": "(Impostors):\nAs the Exorcist, you can use /ex in meeting to kill a player who speaks in the next set amount of seconds after using the command. Depending on the settings, the Exorcist can kill multiple people in one meeting. You'll lose your ability if you excersise too many people.",